### DISCUSSION TITLE
classes/class_audioserver

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#273] [User-contributed notes]
# classes/class_audioserver

Inherits: Object Server interface for low-level audio access. Description: AudioServer is a low-level server interface for audio access. It is in charge of creating sample data (playable audio) as ...

https://docs.godotengine.org/en/stable/classes/class_audioserver.html

**Comment by npnorman** [comment]
void set_bus_mute(bus_idx: int, enable: bool) 
If true, the bus at index bus_idx is muted.

Note: It also un-mutes if the bool is `false`

**Comment by Sithoid** [comment]
Note: get_bus_effect_instance() does NOT return the AudioEffectInstance class, but rather whichever inherited class it fetched. There are as many AudioEffectInstance classes as there are AudioEffect classes, although they are omitted in this class reference. If you're after an AudioEffectSpectrumAnalyzerInstance and chose the wrong bus/effect index, you might as well receive the undocumented AudioEffectEQInstance or the likes.
